lib/spack/llnl/util/filesystem.py

```diff
@@ -221,12 +222,29 @@ def mime_type(filename):
     Returns:
         Tuple containing the MIME type and subtype
     """
+    if sys.platform == "win32":
+        tty.warn("File not available on Windows")
+        return None, None
```

Comment:
Is there a reason we don't let this just delve into `_get_mime_type` and fail with a `RuntimeError`?

Reply:
I wrote this first and was not confident about my `RuntimeError` choice. Will amend.